pkg/istio-agent/xds_proxy_delta.go

```diff
@@ -282,7 +296,7 @@ func (p *XdsProxy) deltaRewriteAndForward(con *ProxyConnection, resp *discovery.
 		resp.Resources[i].Resource = resources[i]
 	}
 
-	proxyLog.Debugf("forward ECDS resources %+v", resp.Resources)
+	proxyLog.WithLabels("resources", slices.Map(resp.Resources, (*discovery.Resource).GetName), "removals", resp.RemovedResources).Debugf("forward ECDS")
```

Comment:
Nit:  On line 220, RemovedResources in the proxyLog has a label = "removes" and a label value = len(resp.RemovedResources).

On line 299, RemovedResources in the proxyLog has a different label = "removals" and a different label value = resp.RemovedResources. 

Consider using a consistent label and value for RemovedResources in the proxyLog, e.g., label="removedresources", lable value=resp.RemovedResources.

Reply:
Thanks, I updated to `removes`

Reply:
Thanks. Why does the label "removes" on line 220 have a value=len(resp.RemovedResources) whereas the label "removes" on line 299 has a value=resp.RemovedResources? If only logging the len, the label "removes" on line 299 should have a value=len(resp.RemovedResources) too. If logging the resp.RemovedResources itself, the label "removes" on line 220 should have a value=resp.RemovedResources.

Reply:
For ECDS we know there are likely only a few values, but others have O(10k) so it would be very noisy to add them all. I can make this `len` if you want though

Reply:
A consistent semantic for a label will improve code readability. As resp.RemovedResources may contain a large number of strings, it sounds good to me to log the len of resp.RemovedResources on line 220 and line 299.